prep-css-display/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Display Properties</title>
+  <style>
+
+  </style>
+</head>
+
+<body>
+  <div style="border: 4px solid blue;">I am a div and by default I am a block, which means I take up all the space I can horizontally.</div>
+  <div>
+    I am a div but I have <span style="border: 4px solid green; height: 100px; width: 100px;">span elements inside me</span>.
+    Span elements are <span style="border: 4px solid green; height: 100px; width: 100px;">display inline by default</span>,
+    so they only take up as much as space as they need.
+  </div>
+  <div>
+    <button style="border: 4px solid red;
+height: 40px;
+width: 200px;">Hello!</button>
+    <button style="border: 4px solid red;
+height: 40px;
+width: 200px;">Buttons are</button>
+    <button style="border: 4px solid red;
+height: 40px;
+width: 200px;">Inline Block</button>
+    <button style="border: 4px solid red;
+height: 40px;
+width: 200px;">By Default</button>
+  </div>
+
+  <ul  style="border: 4px solid purple;">
+    <li style="border: 4px solid magenta; display: inline;">
+      Home
+    </li>
+    <li style="border: 4px solid magenta; display: inline;">
+      Products
+    </li>
+    <li style="border: 4px solid magenta; display: inline;">
+      About Us
+    </li>
+    <li style="border: 4px solid magenta; display: inline;">
+      Contact Us
+    </li>
```

Comment:
Hi Kenny, for this exercise, we are putting all of our CSS code inside of a `<style>` element in the document's `<head>`

```html
<!DOCTYPE html>
<html>
<head>
  <title></title>
  <style>

    /* CSS code goes here */

  </style>
</head>
<body>

</body>
</html>
```

Reply:
<img width="965" alt="Screen Shot 2021-08-04 at 1 33 31 PM" src="https://user-images.githubusercontent.com/55222182/128251788-99f1d0a4-28f1-4d2e-b13d-f1b3f99d0f79.png">